HotelPriceScout/Pages/Prices.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
```

Comment:
Den her fil skal lige portes til den klasse, som Sarmi og jeg har lavet. Den hedder RoomTypePrices. Bruges CompareTo() metoden? For så skal vi have den tilføjet.

Reply:
Yes